core/src/main/java/com/scalar/db/common/checker/OperationChecker.java

```diff
@@ -303,10 +307,7 @@ private void checkColumnsInPut(Put put, TableMetadata metadata) {
     for (Column<?> column : put.getColumns().values()) {
       if (!new ColumnChecker(metadata, false, false, false, true).check(column)) {
         throw new IllegalArgumentException(
-            "The column value is not properly specified. Invalid column: "
-                + column
-                + ", Operation: "
-                + put);
+            CoreError.OPERATION_CHECK_ERROR_TABLE_NOT_FOUND.buildMessage(column, put));
```

Comment:
```suggestion
            CoreError.OPERATION_CHECK_ERROR_INVALID_COLUMN.buildMessage(column, put));
```

Reply:
I think the number of arguments was wrong, but the CI didn't fail...?

Reply:
Oops... Good catch, thank you.

> I think the number of arguments was wrong, but the CI didn't fail...?

Ah, it seems like the error occurs only when the number of arguments is less than the number of the parameters. Umm...